Sources/SentryCrash/Recording/Tools/SentryHook.c

```diff
@@ -176,39 +201,19 @@ sentrycrash_install_async_hooks(void)
     }
 
     sentrycrash__hook_rebind_symbols(
-        (struct rebinding[1]) {
+        (struct rebinding[6]) {
```

Comment:
I wonder if there was some obscure reason the code was done the way it was.

Reply:
lol, just me copy-pasting rather than passing a real list :-D So this is an unrelated driveby refactor.